apps/frontend/src/app/pages/dashboard/community/community-news/community-news.component.ts

```diff
@@ -1,8 +1,55 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { MomBlogService } from '@momentum/frontend/data';
+import { finalize } from 'rxjs/operators';
+import * as txml from 'txml';
 
 @Component({
   selector: 'mom-community-news',
   templateUrl: './community-news.component.html',
-  styleUrls: ['./community-news.component.scss']
+  styleUrls: ['./community-news.component.scss'],
+  encapsulation: ViewEncapsulation.None
 })
-export class CommunityNewsComponent {}
+export class CommunityNewsComponent implements OnInit {
+  // The number of blog posts to display on this component
+  private readonly POSTS_DISPLAYED = 10;
+
+  feed: any;
```

Comment:
May as well type this up a bit (though we don't need to care about the other crap besides `<item>`s. Looks like could be 
```ts
this.feed: { item: BlogPost[]; } = {};
```